backend/indexer/indexer.py

```diff
@@ -259,29 +259,6 @@ async def ingest_data_points(
             documents_to_be_upserted.append(chunk)
         logger.info("%s -> %s chunks", loaded_data_point.local_filepath, len(chunks))
 
-        # delete the file from temp dir after processing
-        try:
-            if loaded_data_point.local_filepath:
-                os.remove(loaded_data_point.local_filepath)
-                logger.debug(
-                    f"Processing done! Deleting file {loaded_data_point.local_filepath}"
-                )
-        except Exception as e:
-            logger.exception(
-                f"Failed to delete file {loaded_data_point.local_filepath} after processing. Error: {e}"
-            )
-        # delete the local_filepath from the loaded_data_point object
-        try:
-            if loaded_data_point.local_metadata_file_path:
-                os.remove(loaded_data_point.local_metadata_file_path)
-                logger.debug(
-                    f"Processing done! Deleting file {loaded_data_point.local_metadata_file_path}"
-                )
-        except Exception as e:
-            logger.exception(
-                f"Failed to delete file {loaded_data_point.local_metadata_file_path} after processing. Error: {e}"
-            )
-
```

Comment:
Why was this removed?

Reply:
The temp directory from the library is auto-cleaned when the variable is out of scope. So, this code was redundant.